Vorstellung/README.md

```diff
@@ -0,0 +1,164 @@
+# Ein einfacher Pandoc/Markdown-Spickzettel
+
+pandoc ist ein Toolpaket zur Konvertierung zwischen Textformaten. Als
+Schriftsteller (und nicht als Techniker) finde ich es nützlich, um
+etwas, das in einfachem Text geschrieben wurde, in eine RTF-Datei zu
+konvertieren, die ich in einem Textverarbeitungsprogramm - bis hin zum
+gefürchteten Word - problemlos mit einigen Formatierungen lesen kann.
+Was ich hier beschreibe, ist nur ein winziger Bruchteil dessen, was man
+damit machen kann.
+
+Es ist so einfach wie:
+
+`$ pandoc -s datei.txt -o datei.rtf`
+
+(Ersetzen Sie \"rtf\" durch html usw., um etwas anderes zu erhalten).
+
+Es kann eine Datei konvertieren, die nur aus Tippen besteht, aber wenn
+Sie ein wenig einfache Formatierung verwenden, ziemlich genau eine
+Version der Markdown-Syntax, können Sie ein schönes RTF mit einigen
+nützlichen Zusatzfunktionen (zum Beispiel Kursivschrift) erhalten. Für
+eine Überschrift unterstreichen Sie mit Gleichheitszeichen oder
+Bindestrichen in der nächsten Zeile;
+
+    Überschrift hier
```

Comment:
Würde ich vielleicht raus nehmen und nur die `# Heading` syntax mit reinnehmen, um eine konsistente syntax in den ganzen geschriebenen Dokumenten vorzugeben.